### DISCUSSION TITLE
Guide for compiling Leap SDK

### DISCUSSION THREAD

**Discussion by ghost** [seanschneeweiss/RoSeMotion#16] [General]
Kia ora, 
I would like to use the Leap SDK with python3.10 but haven't been able to find any up to date guides. 
Would you be interested in creating a guide to help others compile the SDK or do you know of any recent guides created by other people?

Thanks, Omelia

**Comment by seanschneeweiss** [comment]
Hello Omelia,

thank you for starting the discussion.

I was mostly following the instructions from https://github.com/leapmotion/LeapCxx and adding minor changes to the cmake files to make them work with the selected CMAKE and SWIG versions.
https://github.com/seanschneeweiss/RoSeMotion/tree/master/app/resources/LeapSDK/v53_python39

See 
https://github.com/seanschneeweiss/RoSeMotion/blob/master/app/resources/LeapSDK/CMakeLists.txt and https://github.com/seanschneeweiss/RoSeMotion/blob/master/app/resources/LeapSDK/src-CMakeLists.txt.

When I should get some time this year to build some newer SDK I will make sure to document the process.
